dll/win32/xolehlp/xolehlp.c

```diff
@@ -637,8 +637,10 @@ BOOL WINAPI DllMain( HINSTANCE hinst, DWORD reason, LPVOID reserved )
 
     switch (reason)
     {
+#ifndef __REACTOS__
         case DLL_WINE_PREATTACH:
             return FALSE;    /* prefer native version */
+#endif
```

Comment:
This change (everywhere) looks good to me, though I suggest to move it to a separate PR (doable now).
And to comment out `DLL_WINE_PREATTACH` definition in `winnt_old.h`, with an updated comment.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=DLL_WINE_PREATTACH

Reply:
#7802